pkg/kv/kvserver/batcheval/cmd_export_test.go

```diff
@@ -436,42 +435,38 @@ func TestExportRequestWithCPULimitResumeSpans(t *testing.T) {
 
 	ctx := context.Background()
 	rng, _ := randutil.NewTestRand()
-	tc := testcluster.StartTestCluster(t, 3, base.TestClusterArgs{
```

Comment:
It'd be good to get @stevendanna to sign off on this test going from multi-node cluster to single server. If it's acceptable, then we should remove the SCATTER command as well as the scatter setting override since it's meaningless in single node setup.

Reply:
I think we need to keep this as a multi-node test. Part of what this is testing is that the DistSender is able to correctly decide whether or not a request that needs to be split across multiple ranges can be parallelised and combined or not.